components/LinkCustom/LinkCustom.test.tsx

```diff
@@ -5,11 +5,11 @@ import LinkCustom from './LinkCustom';
 describe('LinkCustom Component', () => {
   it('renders with default props', () => {
     render(
-      <LinkCustom children="Test link" href="https://example.com"></LinkCustom>,
+      <LinkCustom children="Test link" data-testid="testLink" href="https://example.com"></LinkCustom>,
```

Comment:
This should follow our custom classes and other `data-testid` and be hyphenated, not camelCased.